local/gugcat/locallib.php

```diff
@@ -277,6 +290,16 @@ public static function update_grade($userid, $itemid, $grade, $notes = null, $gr
 
     public static function convert_grade($grade){
         $scale = self::$GRADES;
+
+        //add admin grades in scale
+        $scale[NON_SUBMISSION] = NON_SUBMISSION_AC;
```

Comment:
I think you can call the $AGGRADE array from grade_aggregation class here, then just concatenate the NS.